lib/UI/UI.rb

```diff
@@ -54,9 +63,12 @@ def announce message
   end
 
   def announce_results(board, players)
-    announce("It's a tie") if board.tie?
+    if board.win?(players)
     players.each do |player|
-      announce(player.symbol + " has won!") if board.is_a_winner?(player)
```

Comment:
Idiomatic ruby usually doesn't have the `is`, `has`, etc. in the method name. That's expressed via the question mark.